client/src/nls/zh/locale.js

```diff
@@ -180,54 +187,55 @@ define({
 
     // ---- hda-edit
     "Edit attributes": "编辑属性",
-    "Cannot edit attributes of datasets removed from disk": "无法编辑已从磁盘中删除的数据集的属性",
+    "Cannot edit attributes of datasets removed from disk": "无法编辑已从磁盘中移除的数据集的属性",
     "Undelete dataset to edit attributes": "取消删除数据集以编辑属性",
-    "This dataset must finish uploading before it can be edited": "该数据集必须先上传完成, 然后才能编辑",
+    "This dataset must finish uploading before it can be edited": "该数据集必须先上传完成, 才能编辑",
     "This dataset is not yet editable": "该数据集不可编辑",
 
     Delete: "删除",
     "Dataset is already deleted": "数据集已被删除",
 
-    "View or report this error": "查看或报告此错误",
+    "View or report this error": "查看或报告当前错误",
 
-    "Run this job again": "重新运行此任务",
+    "Run this job again": "重新运行当前任务",
 
     Visualize: "可视化",
     "Visualize in": "的可视化",
 
     "Undelete it": "取消删除",
-    "Permanently remove it from disk": "从磁盘中永久删除",
+    "Permanently remove it from disk": "从磁盘中永久移除",
     "Unhide it": "取消隐藏",
 
-    "You may be able to": "您可能可以",
-    "set it manually or retry auto-detection": "手动设置或重试自动检测",
+    "You may be able to": "您可以",
+    "set it manually or retry auto-detection": "手动设置或重新执行自动检测",
 
     "Edit dataset tags": "编辑数据集标签",
     "Edit dataset annotation": "编辑数据集备注",
 
     "Tool Help": "工具帮助",
 
     // ---------------------------------------------------------------------------- admin
-    "Search Tool Shed": "搜索 Tool Shed",
-    "Monitor installing repositories": "",
-    "Manage installed tools": "管理已安装的工具",
-    "Reset metadata": "重置元数据",
-    "Download local tool": "下载本地工具",
+    "Reset passwords": false,
+    "Search Tool Shed": false,
+    "Monitor installing repositories": false,
+    "Manage installed tools": false,
+    "Reset metadata": false,
+    "Download local tool": false,
     "Tool lineage": false,
-    "Reload a tool's configuration": "重新加载工具的配置",
-    "Review tool migration stages": "查看工具迁移状态",
-    "View Tool Error Logs": "查看工具错误日志",
-    "Manage Allowlist": "管理显示白名单",
-    "Manage Tool Dependencies": "管理工具依赖",
-    Users: "用户",
-    Groups: "组别",
+    "Reload a tool's configuration": false,
```

Comment:
@shenweiyan is there a reason you removed this translation?

Reply:
As an administrators, I think the English interface will be better. And i18n processing is just to make it more convenient for non-administrators to use galaxy.